CONTRIBUTING.md

```diff
@@ -28,6 +28,10 @@ issues first, as it's possible that someone else reported the same issue before
 you. Though it helps save time, don't worry! We won't mind if you accidentally
 post a duplicate issue.
 
+Please note that if an issue can only be reproduced on a nightly compiler it will be closed and marked "Will not fix."
```

Comment:
I guess I want to set the expectation that Amethyst is not going to change its codebase to accommodate rust compiler bugs on beta/nightly that will disappear in a few weeks.  The language used specifically to accomplish that could be improved.